docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -31,6 +31,12 @@ Using a policy that makes use of the <<ilm-rollover, rollover>> action
 in the hot phase will avoid this situation and the need for a manual rollover for future
 managed indices.
 
+NOTE: Mounting and relocating the shards of {search-snap} indices involves
+copying data from the snapshot repository. This may incur different costs from
+the copying between nodes in your cluster that happens with regular indices.
+These costs are typically lower, but in some environments they may be higher.
+See <<searchable-snapshots-costs>> for more details.
```

Comment:
If you use my other suggestion, you can reuse some of that content here.

Either way, I'd consider moving this up the page (right after the paragraph ending `... corresponding data tier.`). I'd also consider removing the other two `IMPORTANT` and `NOTE` admonitions on the page. Having a stack of admonitions made it more likely for me to glaze over each.

```suggestion
[IMPORTANT]
====
include::{es-repo-dir}/searchable-snapshots/index.asciidoc[tag=search-snap-costs]
See <<searchable-snapshots-costs>>.
====
```